src/includes/control-cmds.h

```diff
@@ -1,148 +1,152 @@
 #ifndef DINIT_CONTROL_CMDS_H_INCLUDED
 #define DINIT_CONTROL_CMDS_H_INCLUDED 1
 
+#include "control-datatypes.h"
+
 // Dinit control command packet types
 
 // Requests:
+enum class cp_cmd :dinit_cptypes::cp_cmd_t {
```

Comment:
formatting nit: can you put spaces around `:` to match style elsewhere? i.e
```
enum class cp_cmd : dinit_cptypes::cp_cmd_t {
```

Reply:
done.